napari/components/_tests/test_viewer_model.py

```diff
@@ -1044,32 +1056,14 @@ def test_get_status_text():
     )
     viewer.tooltip.visible = False
     assert viewer._calc_status_from_cursor() == (
-        {
-            'coordinates': ' [1 2]: 0; a: 1',
-            'layer_base': 'Labels',
-            'layer_name': 'Labels',
-            'plugin': '',
-            'source_type': '',
-        },
+        ' [1 2] » Labels: 0; a: 1    ',
```

Comment:
I don't like this change. I prefer to return even richer dict and construct the string in the status bar.